plone/app/registry/tests/test_exportimport.py

```diff
@@ -393,6 +394,29 @@ def test_import_value_only(self):
             self.registry['test.export.simple']
         )
 
+    def test_import_value_validation(self):
+        xml = """\
+<registry>
+    <record name="test.export.simple">
+        <value></value>
+    </record>
+</registry>
+"""
+        context = DummyImportContext(self.site, purge=False)
+        context._files = {'registry.xml': xml}
+
+        self.registry.records['test.export.simple'] = \
+            Record(field.TextLine(title=u"Simple record", default=u"N/A"),
+                   value=u"Sample value")
+
+        try:
+            importRegistry(context)
+        except zope.schema._bootstrapinterfaces.RequiredMissing:
+            # this should raise RequiredMissing
```

Comment:
This can be done with:

```
with self.assertRaises(zope.schema._bootstrapinterfaces.RequiredMissing):
    importRegistry(context)
```

Reply:
Thank's for the hint ... I'll change that